ee/vellum_ee/workflows/tests/test_serialize_module.py

```diff
@@ -14,3 +14,33 @@ def test_serialize_module_happy_path():
     assert "workflow_raw_data" in result.exec_config
     assert "input_variables" in result.exec_config
     assert "output_variables" in result.exec_config
+
+
+def test_serialize_module_includes_additional_files():
+    """Test that serialize_module includes additional files from the module directory."""
+    module_path = "tests.workflows.module_with_additional_files"
+
+    result = BaseWorkflowDisplay.serialize_module(module_path)
+
+    assert hasattr(result, "exec_config")
+    assert hasattr(result, "errors")
+    assert isinstance(result.exec_config, dict)
+    assert isinstance(result.errors, list)
+
+    assert "module_data" in result.exec_config
+    assert "additional_files" in result.exec_config["module_data"]
+
+    additional_files = result.exec_config["module_data"]["additional_files"]
```

Comment:
can you add assertions that the workflow.py file and none of the `node` files are in this mapping?